lib/screens/settings/about.dart

```diff
@@ -0,0 +1,111 @@
+                      ),
+                    ),
+                  ],
+                ),
+              ),
+              Card(
+                child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: ExpandablePanel(
+                    header: Text(
+                      "What is the Mentorship System",
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        fontSize: 22,
+                      ),
+                    ),
+                    collapsed: Text("Mentorship System is a cross platform application"),
+                    expanded: Text(
+                      "Mentorship System is a cross platform application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period of time.",
+                    ),
+                  ),
+                ),
+              ),
+              Card(
+                child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: ExpandablePanel(
+                    header: Text(
+                      "Contributing to Mentorship",
+                      style: TextStyle(
+                        fontWeight: FontWeight.bold,
+                        fontSize: 22,
+                      ),
+                    ),
+                    collapsed: Text("Head on over to our github to contribute"),
+                    expanded: Container(
+                      height: 40,
+                      width: 100,
+                      child: RaisedButton(
+                        color: Color.fromRGBO(36, 41, 46, 1),
+                        child: Row(
+                          children: <Widget>[
+                            Image.asset(
+                              "assets/images/GitHub-Mark-Light-120px-plus.png",
+                              width: 20,
+                              height: 25,
+                            ),
+                            SizedBox(
+                              width: 5,
+                            ),
+                            Text(
+                              "Github",
+                              style: TextStyle(color: Colors.white),
+                            ),
+                          ],
+                        ),
+                        onPressed: _launchURL,
+                      ),
+                    ),
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+_launchURL() async {
+  const url = 'https://github.com/anitab-org/mentorship-flutter';
+  if (await canLaunch(url)) {
+    await launch(url);
+  } else {
+    throw 'Could not launch $url';
```

Comment:
When can it fail and what if it happens? Maybe just `print()` it and show an AlertDialog saying that it's impossible, instead of just throwing an exception.